### PR TITLE
Added:- "view all packages from the author"

### PR DESCRIPTION
**Pull request by iammuhammadahmad**
Hi!
Added the new functionality "view all packages from author". 
Please have a look at this.
Thanks,

### CODE REVIEW

File: action/index.js [hunk 1]
```diff
@@ -1,95 +1,22 @@
-module.exports =
```
Comment (twodayslate) [inline action/index.js:1]: Why was this updated?

File: views/userAllPackage.ejs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<form class="form my-3" action="#" method="GET">
+	<div class="input-group">
+    <input class="form-control form-control-lg" type="text" placeholder="Search Swift Package Registry" id="term" name="term">
+    <span class="input-group-append">
+    	<button type="submit" class="btn btn-primary" disabled>Search</button>
+    	</span>
+    </input>
+    </div>
+</form>
+<h2>Swift Packages</h2>
```
Comment (twodayslate) [inline views/userAllPackage.ejs:12]: Set title to "`<Username>`'s Swift Packages"

File: views/userAllPackage.ejs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+		<td><%= package.error || package.info.description %></td>
+	</tr>
+  <% }); %>
+  </tbody>
+</table>
+
+<%} else{ %>
+	<div>
+		<h1 class="text-center text-danger" >404</h1>
```
Comment (twodayslate) [inline views/userAllPackage.ejs:39]: Can the route handle this?

res.render('404')

File: lib/userAllPackage.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+module.exports = function (router) {
+  router.get('/:owner', async function(req, res) {
+    var packages = await router.db.Package.findAll({
+      where: {
+        'info.owner.login': req.params.owner
+      }})
+    res.render('userAllPackage', { packages: packages,title: req.params.owner + "All Packages" })
```
Comment (twodayslate) [inline lib/userAllPackage.js:7]: Title can be `req.params.owner + "'s Packages"`?

**Review by twodayslate — CHANGES_REQUESTED** [review]

File: views/userAllPackage.ejs (hunk 3 above)
Comment (iammuhammadahmad) [reply views/userAllPackage.ejs:39]: Yes, It is handling. if(packages.length >0) does not true then below code will be executed. It's means the user's has not packages added so the user is not exist. i.e( http://localhost:3000/fastlanessafs). in this case **fastlanessafs** does not exist. 
<%} else{ %>
	<div>
		<h1 class="text-center text-danger" >404</h1>
		<h4 class="text-center"><strong>oOPs! </strong>PAGE NOT FOUND</4>
	</div
<%}%>

Comment (twodayslate) [reply views/userAllPackage.ejs:39]: If the user doesn't exist then it should 404 in the route code then - not the template.

File: lib/userAllPackage.js (hunk 4 above)
Comment (iammuhammadahmad) [reply lib/userAllPackage.js]: Done this one.

File: views/userAllPackage.ejs (hunk 2 above)
Comment (iammuhammadahmad) [reply views/userAllPackage.ejs:12]: Done this one.

**Pull request closed without merging**